### PR TITLE
Fix README export documentation

### PR DESCRIPTION
**Pull request by slacktracer**
I updated the readme to reflect this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Then with a module bundler like [Rollup](https://rollupjs.org) or [Webpack](http
 
 ```js
 import { app, h } from 'hyperapp';
-import debug from 'hyperapp-debug';
+import { debug } from 'hyperapp-debug';
 ```
 
 If you don't want to set up a build environment, you can download Hyperapp Debug from a CDN like [unpkg.com](https://unpkg.com/hyperapp-debug) and it will be globally available through the <samp>window['hyperapp-debug'].default</samp> object. We support all ES5-compliant browsers, including Internet Explorer 10 and above.
```
Comment (mrozbarry) [inline README.md]: There is a good possibility that this is also wrong, too. I think it is now just `<samp>window.debug</samp>`. I'll need to double check that quickly first, though.

Comment (mrozbarry) [reply README.md]: Ah, now it's `<samp>window.withDebug</samp>`. I've updated my codepens. If you can update the docs in your PR with this, too, that'd be cool 😄

**Comment by slacktracer** [comment]
Hi! I'll try and do it in the weekend. =)

File: README.md (hunk 1 above)
Comment (slacktracer) [reply README.md]: I just did it. =) Is it correct?

**Review by mrozbarry — APPROVED** [review]

**Pull request merged**